solidity/ecdsa/contracts/WalletRegistryGovernance.sol

```diff
@@ -447,6 +542,33 @@ contract WalletRegistryGovernance is Ownable {
         newSubmitterPrecedencePeriodLength = 0;
     }
 
+    /// @notice Get the time remaining until the minimum authorization amount
+    ///         can be updated.
+    /// @return Remaining time in seconds.
+    function getRemainingMimimumAuthorizationUpdateTime()
+        external
+        view
+        returns (uint256)
+    {
+        return
+            getRemainingChangeTime(
+                minimumAuthorizationChangeInitiated,
+                CRITICAL_PARAMETER_GOVERNANCE_DELAY
+            );
+    }
+
```

Comment:
Can we add a comment for this function as well?

Reply:
Yes, good catch! Please see https://github.com/keep-network/keep-core/pull/2848/commits/c68ce675c83f6061a92380a3f3723dbaa1e1c91b